SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -13,15 +13,23 @@
 
 #include <WorkerSDK/improbable/c_worker.h>
 
+DEFINE_LOG_CATEGORY_STATIC(LogConnectionConfig, Log, All);
```

Comment:
This should only be used in source files, there is an _EXTERN variant which allows you to declare where the definition goes - this currently will create a static in every translation unit 

<header>
DECLARE_LOG_CATEGORY_EXTERN

<cpp>
DEFINE_LOG_CATEGORY

Reply:
There isn't a corresponding .cpp for this header (which is why I used DEFINE_LOG_CATEGORY_STATIC)

Reply:
So this is a bit weird in c++, it will define a version of this static in every translation unit (cpp) including this file and will be stored in the binary more times than desired. Essentially every cpp will have their own version of this static variable which isn't shared. I just checked and this file is indrectly included in 97 cpps :D might be worth just creating the cpp to avoid this

Reply:
and potentially issues in unity builds for multiple definitions which is a more solid case for a cpp